apps/store/extensions/app/social-reviews/themes/store/js/lib/messages.js

```diff
@@ -0,0 +1,103 @@
+var messages = {};
+messages.alertSuccess = function(value){
```

Comment:
Thank you, I have fixed this issue in following [commit](https://github.com/wso2/carbon-store/pull/643/commits/358a87f6fa0ffb067480daf1fe3e463bed87080e)